api/xml/libero/extensions/doi.rng

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<grammar xmlns="http://relaxng.org/ns/structure/1.0" ns="http://libero.pub">
+
+    <define name="libero.article.content" combine="interleave">
+        <optional>
```

Comment:
Can't see a way to make these extensions either optional or required, without having an extension for each (probably using a shared definition for the actual elements/attributes).